var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -84,6 +84,34 @@ class Geant4(CMakePackage):
     variant("timemory", default=False, description="Use TiMemory for profiling", when="@9.5:")
     variant("vtk", default=False, description="Enable VTK support", when="@11:")
 
+    # For most users, obtaining the Geant4 data via Spack will be useful; the
+    # sticky, default-enabled `+data` variant ensures that this happens.
+    # Furthermore, if this variant is enabled, Spack will automatically set the
+    # necessary environment variables to ensure that the Geant4 code runs
+    # correctly.
+    #
+    # However, the Geant4 data is also large and it is, on many machines used
+    # in HEP, already available via e.g. CVMFS. In these cases, users can save
+    # network bandwidth by using externally supplied Geant4 data. This can be
+    # done in two different ways.
+    #
+    # The first is to declare the Geant4 data directories as externals. This
+    # can be done by manually adding them to the `packages.yaml` file, e.g.:
+    #
+    # ```
+    # g4radioactivedecay:
+    #   externals:
+    #   - spec: g4radioactivedecay@5.6
+    #     prefix: /cvmfs/geant4.cern.ch/
```

Comment:
This prefix isn't correct, as Spack expects each dataset to be installed under the prefix as:

```
+- <prefix>/
   +- share/
      +- data/
         +- <DATATSET_NAME><VERSION>
```

The CVMFS repo is setup to support that right now, so should just document a generic prefix, and note the structure spack expects.

Reply:
I may be misunderstanding, but how is this not correct? E.g.:

```console
$ ls -l /cvmfs/geant4.cern.ch/share/data/G4EMLOW6.42
total 152
drwxr-xr-x. 2 cvmfs cvmfs  4096 Oct 22  2014 auger
drwxr-xr-x. 2 cvmfs cvmfs  4096 Oct 22  2014 brem
drwxr-xr-x. 2 cvmfs cvmfs  4096 Oct 22  2014 brem_SB
...
```

This matches the schema you described, right? Although indeed it may be better to provide more general instructions.

Reply:
Updated the documentation to be more generic; thanks!